test/e2e/curated_packages_test.go

```diff
@@ -187,17 +190,19 @@ func runCuratedPackageInstallSimpleFlow(test *framework.ClusterE2ETest) {
 // decisions about test packages or methodologies, right now.
 
 func TestCPackagesDockerUbuntuKubernetes120SimpleFlow(t *testing.T) {
+	framework.CheckCuratedPackagesCredentials(t)
 	test := framework.NewClusterE2ETest(t,
 		framework.NewDocker(t),
 		framework.WithClusterFiller(api.WithKubernetesVersion(v1alpha1.Kube120)),
-		framework.WithPackageConfig(t, packageBundleURI(v1alpha1.Kube120),
+		framework.WithPackageConfig(t, packageBundleURI(v1alpha1.Kube121),
```

Comment:
Is this correct? Why is this changed to Kube121 when the test is for Kube120?

Reply:
fixed